proto/cacheclient.proto

```diff
@@ -31,6 +32,12 @@ message _GetResponse {
   string message = 3;
 }
 
+message _DeleteRequest {}
```

Comment:
I am just curious here.

I would have thought the delete here means that we want to remove an entry from the cache. Is there a `bytes cache_key = 1` needed here?

Reply:
oops yes I removed the wrong one 😅 . good catch